packages/datepickers/demo/datepicker.stories.mdx

```diff
@@ -25,6 +30,17 @@ import README from '../README.md';
         control: 'radio',
         options: DATE_STYLE_OPTIONS,
         table: { category: 'Story' }
+      },
+      hasMessage: { name: 'Message', control: { type: 'boolean' }, table: 'Story' },
+      message: { name: 'children', control: { type: 'text' }, table: { category: 'Message' } },
+      validation: {
+        options: ['success', 'warning', 'error'],
+        control: { type: 'radio' },
+        table: { category: 'Input' }
+      },
+      validationLabel: {
+        control: { type: 'text' },
+        table: { category: 'Input' }
```

Comment:
```suggestion
        table: { category: 'Message' }
```

[nit] the `validationLabel` actually belongs to the Message's icon. See https://garden.zendesk.com/components/input#message

Reply:
Ah, shoot, I see it now in the Input story too. Fixing.